docs/src/pages/layout/grid/grid.md

```diff
@@ -23,8 +23,11 @@ The grid system is implemented with the `Grid` component:
 ## Spacing
 
 The responsive grid focuses on consistent spacing widths, rather than column width.
-Material design margins and columns follow an **8dp** square baseline grid.
-Spacing can be 8, 16, 24, 32 or 40dp wide.
+Material design margins and columns follow an **8px** square baseline grid.
+The spacing property is an integer in the [0, 8] interval.
+By default, the spacing between two grid items follows a linear function: `output(spacing) = spacing * 8px`, e.g. `spacing={2}` creates a 16px wide gap.
+
+This output transformation function can be customized using the theme, exactly like [the system spacing](/system/spacing/#transformation).
```

Comment:
Is it "exactly like" the system spacing (but a different key in the theme), or actually the system spacing?

Reply:
I'm going with 
`This output transformation function can be customized [using the theme](/customization/themes/#spacing).`